changelog.d/772.bugfix

```diff
@@ -0,0 +1 @@
+Fix swallowed text between 2 mentions
```

Comment:
@akirk This line needs to end with a '.' or '!' as per tests.

Reply:
Thanks, added !